packages/core/test/client.test.js

```diff
@@ -27,45 +25,53 @@ describe('@bugsnag/core/client', () => {
   })
 
   describe('use()', () => {
-    it('supports plugins', done => {
-      const client = new Client({ apiKey: '123' })
-      client.use({
-        name: 'test plugin',
-        description: 'nothing much to see here',
-        init: (c) => {
-          expect(c).toEqual(client)
-          done()
-        }
+    it('supports plugins', () => {
+      return new Promise(resolve => {
+        const client = new Client({ apiKey: '123' })
+        client.use({
+          name: 'test plugin',
+          description: 'nothing much to see here',
+          init: (c) => {
+            expect(c).toEqual(client)
+            resolve()
+          }
+        })
       })
     })
   })
 
   describe('logger()', () => {
-    it('can supply a different logger', done => {
-      const client = new Client({ apiKey: 'API_KEY_YEAH' })
-      const log = (msg) => {
-        expect(msg).toBeTruthy()
-        done()
-      }
-      client._logger = { debug: log, info: log, warn: log, error: log }
-      client._logger.debug('hey')
+    it('can supply a different logger', () => {
+      return new Promise(resolve => {
+        const client = new Client({ apiKey: 'API_KEY_YEAH' })
+        const log = (msg) => {
+          expect(msg).toBeTruthy()
+          resolve()
+        }
+        client._logger = { debug: log, info: log, warn: log, error: log }
+        client._logger.debug('hey')
+      })
     })
-    it('can supply a different logger via config', done => {
-      const log = (msg) => {
-        expect(msg).toBeTruthy()
-        done()
-      }
-      const client = new Client({
-        apiKey: 'API_KEY_YEAH',
-        logger: {
-          debug: log,
-          info: log,
-          warn: log,
-          error: log
+    it('can supply a different logger via config', () => {
+      return new Promise(resolve => {
+        const log = (msg) => {
+          expect(msg).toBeTruthy()
+          resolve()
         }
+        const client = new Client({
+          apiKey: 'API_KEY_YEAH',
+          logger: {
+            debug: log,
+            info: log,
+            warn: log,
+            error: log
+          }
+        })
+        client._logger.debug('hey')
       })
-      client._logger.debug('hey')
     })
+
+    // eslint-disable-next-line jest/expect-expect
```

Comment:
I think it would be better to add an assertion to this test rather than disable the lint error